float64	tl_spd
float64	tr_spd
float64	bl_spd
float64	br_spd
float64	tl_pos
float64	tr_pos
float64	bl_pos
float64	br_pos